style.css

```diff
@@ -27,54 +27,56 @@ body {
 ================================*/
 form {
   background: #2c3e50;
-  height: 2.3em; 
+  height: 2.3em;
   width: 100%;
+  display: flex;
```

Comment:
My thoughts here is that what we're aiming to teach/help with in this project is the combination of hapi/socket.io/redis that brought you to it in the first place @yurm04!

Whilst we *want* to make *all* of our code and examples as accessible as possible, I think that it's not the *main* aim of this repo, so it shouldn't be a blocker for merging these excellent changes in.  There are lots of improvements to the CSS that could be done if we were being picky (e.g. over-usage of ID selectors and descendent selectors)!

**Suggested next steps** (happy for this to be in a different PR): 
+ Add a line to the readme explaining that we're using `flexbox` for simplicity given the focus of this repo is on the hapi/socket.io/redis combination, but raising a warning this will not play well with anything under IE11 (sadly still under heavy use)
+ Add an issue to the backlog for the more universal solution you've suggested @yurm04 so that it's captured as something to be dealt with when you or anyone else has a little time :blush: